kaggle_environments/envs/kore_fleets/kore_fleets.py

```diff
@@ -146,17 +146,15 @@ def attacker_agent(board):
 
 
 @board_agent
-def simp_agent(board):
+def balanced_agent(board):
     me = board.current_player
-    if board.step % 100  == 0 and me.id == 0:
```

Comment:
Is this deletion also intended? If so could you add a note to the PR description about what it's doing?

Reply:
Yeah leftover from some print statement debugging